auto3dseg/tests/test_algo_templates.py

```diff
@@ -47,14 +48,22 @@
 
 sys.path.insert(0, algo_templates)
 
+num_epochs = 1
+n_data = 8
+num_gpus = 4 if torch.cuda.device_count() > 4 else torch.cuda.device_count()
+num_images_per_batch = 2
+num_epochs_per_validation = 1
+num_iterations = int(num_epochs * n_data / num_images_per_batch / num_gpus)
+len_train_loader = int(n_data / num_gpus / num_images_per_batch)
+num_iterations_per_validation = len_train_loader * num_epochs_per_validation
```

Comment:
hi @mingxin-zheng, what is the purpose for this change?

Reply:
Overall, this is to enhance the test.

With the previous code, some of the algorithms will fail when the `num_epochs` is set to 1, because the `num_iterations` is too small.

After the fix, the code will still work with small `num_epochs` and `num_iterations`. So that's why we have the change.

Reply:
got it. thank you!